blocks/card/card.js

```diff
@@ -33,6 +33,10 @@ export default function decorate(block) {
       { width: '300' },
     ],
   );
+  // adjust style if only 1 card in the block
+  if (block.children.length === 1) {
+    block.closest('.card-container').classList.add('single');
+  }
```

Comment:
I suggest to remove this, and instead on https://github.com/hlxsites/24life/blob/19c1581d70ccde979cb566db743f929d033a1276/blocks/card/card.css#L62-L70 make default behavior to be `display:block`, and then add a `three-columns` (and 4 if you are already at it). 
Like this we don't have multiple ways to configure the columns of a block, and a reasonable default behavior.

Reply:
I switched to columns block. http://localhost:3000/drafts/rrusher/card-columns